test/ETHStakingYieldOracle.t.sol

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.0;
+
+import "forge-std/Test.sol";
+
+import { ETHStakingYieldOracle } from "../src/ETHStakingYieldOracle.sol";
+
+contract ETHStakingYieldOracleTest is Test {
+
+    address bridge        = makeAddr("bridge");
+    address randomAddress = makeAddr("randomAddress");
+
+    ETHStakingYieldOracle oracle;
+
+    event BridgeUpdated(address indexed bridge, bool authorized);
+    event MaxAPRUpdated(uint256 maxAPR);
+    event APRUpdated(uint256 apr);
+
+    function setUp() public {
+        oracle = new ETHStakingYieldOracle();
```

Comment:
transferOwnership to a makeAddr("owener"); address so permissioned calls have to be pranked and notOwner tests don't prank